hxtool/device.py

```diff
@@ -166,6 +180,53 @@ class HX890(HX870):
     nmea_model = HX890NMEAProtocol
 
 
+class GX1400(HX870):
+    """
+    Device object for Standard Horizon GX1400 maritime radios
+    """
+    handle = "GX1400"
+    brand = "Standard Horizon"
+    model = "GX1400"
+    usb_vendor_id = None
+    usb_vendor_name = None
+    usb_product_id = None
+    usb_product_name = None
+    flash_id = ["AM065N"]
+
+    protocol_model = GX1400Protocol
+    config_model = GX1400Config
+    nmea_model = None
+
+    def __init__(self, tty):
+        # Overriding __init_config() in this class is only possible by
+        # using the name _HX870__init_config(), which would be confusing.
+        # Therefore, the constructor definition must be repeated here,
+        # even though it's identical to the one in the superclass.
+        self.tty = tty
+        self.comm = self.protocol_model(tty=tty)
+        self.config = None
+        self.nmea = None
+        self.__init_config()
+
+    def __init_config(self):
```

Comment:
The name choice `__init_config()` instead of `init_config()` makes it harder to override the method.
 
What’s the rationale for using the `__` prefix in these classes? Could we get rid of it?

Reply:
If you need to override, just go ahead and unhide it.

Reply:
I propose to rename `__init_config()` to `init_config()`.

That would allow removing the `__init__()` constructor from this class, as it can then be inherited from the superclass.

Reply:
Try and see how it works out.